soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -677,6 +677,28 @@ void Randomizer::ParseRandomizerSettingsFile(const char* spoilerFileName) {
                             gSaveContext.randoSettings[index].value = 1;
                         }
                         break;
+                    case RSK_ITEM_POOL:
+                        if(it.value() == "Plentiful") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if(it.value() == "Balanced") {
+                            gSaveContext.randoSettings[index].value = 1;            
+                        } else if(it.value() == "Scarce") {
+                            gSaveContext.randoSettings[index].value = 2;
+                        } else if(it.value() == "Minimal") {
+                            gSaveContext.randoSettings[index].value = 3;
+                        }
+                    case RSK_ICE_TRAPS:
+                        if(it.value() == "Off") {
+                            gSaveContext.randoSettings[index].value = 0;
+                        } else if(it.value() == "Normal") {
+                            gSaveContext.randoSettings[index].value = 1;            
+                        } else if(it.value() == "Extra") {
+                            gSaveContext.randoSettings[index].value = 2;
+                        } else if(it.value() == "Mayhem") {
+                            gSaveContext.randoSettings[index].value = 3;
+                        } else if(it.value() == "Onslaught") {
+                            gSaveContext.randoSettings[index].value = 4;
+                        }
```

Comment:
so the reason these never made it into the parsing before is because we never actually need to read these settings in game (they're only used during seed generation to determine what items to place where)

Reply:
Ah okay, would it be better to revert then?

Reply:
nothing wrong with adding them, just noticed you mentioned not knowing why they weren't there before so i figured i'd explain